### PR TITLE
Dilpreet | Test| Tally suggestions and imports in sidebar

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2v1xypr
https://app.clickup.com/t/2v1qdhy

### CODE REVIEW

**Comment by dilpreetsinghofficial** [comment]
Reviewed

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.html [hunk 1]
```diff
@@ -475,29 +475,26 @@ <h3 class="section-head mb-2">{{ localeData?.ecommerce?.shopify_form_heading }}<
...
                         </mat-form-field>
                         <span class="input-group-btn">
-                            <button
-                                mat-button
-                                [disabled]="!apiUrl ? 'disabled' : null"
-                                class="btn share-ledger-button btn-success d-flex align-items-center disabled-btn-color"
-                                (click)="toggleIsCopied()"
+                            <i
+                                class="icon-copy-file"
+                                (click)="urlCopied()"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.html]:
```suggestion
                                (click)="copyUrl()"
```

File: apps/web-giddh/src/app/settings/integration/setting.integration.component.ts [hunk 2]
```diff
@@ -1042,7 +1042,7 @@ export class SettingIntegrationComponent implements OnInit, AfterViewInit {
      *
      * @memberof SettingIntegrationComponent
      */
-    public toggleIsCopied() : void {
+    public urlCopied(): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/settings/integration/setting.integration.component.ts]:
```suggestion
    public copyUrl(): void {
```

**Review by ravinderwalkover — APPROVED** [review]
@dilpreet77 merge only after making suggested changes

**Pull request merged**